app/assets/stylesheets/reaction.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the reaction controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
I usually delete these assets files (The `scss` and `js` files) unless it's clear I need them at present.